packages/inter-protocol/src/auction/scheduler.js

```diff
@@ -209,9 +209,9 @@ export const makeScheduler = async (
   };
 
   // set wakeups for the steps of this round
-  const queueLiveSchedule = () => {
+  const scheduleDescendingSteps = () => {
```

Comment:
this was renamed to "queue" to avoid the overloading of "schedule" (noun for a plan and verb for setting wakeups). ironic that the PR un-overloading "locked" re-overloaded "schedule".  also surprised to see it not mentioned in the commit message.

not a blocker